On OS X, we build the qtimageformats module to get icns support.
But we don't need any of the other image formats, so ensure we
don't build them.

Primarily, we're concerned with webp, which will forcefully include
a potentially outdated libwebp into our binaries and our QImage
handling.

Since icns is the only one we require, only build it.

--- ./qtimageformats/src/plugins/imageformats/imageformats.pro
+++ ./qtimageformats/src/plugins/imageformats/imageformats.pro
@@ -1,10 +1,4 @@
 TEMPLATE = subdirs
-SUBDIRS = \
-#    dds \
-    tga \
-    tiff \
-    wbmp \
-    webp
 
 qtConfig(regularexpression): \
     SUBDIRS += icns
